### PR TITLE
packaging updates

### PR DESCRIPTION
**Pull request by gardnerjr**
* fix case where categories aren't getting defaults if they weren't localized
* added scripts for building just the new package way
* updated buddy build to only build new package
* updated docs

### CODE REVIEW

File: scripts/ProcessTemplates.ps1 [hunk 1]
```diff
@@ -14,26 +14,38 @@ $azureBlobFileNameBase = "community-templates-V2";
 $repoBaseName = "Application-Insights-Workbooks"
 $supportedLanguages = @(
     $defaultLanguage,
-    "cs-cz",
-    "de-de",
+#    "cs-cz",
```
Comment (gardnerjr) [inline scripts/ProcessTemplates.ps1]: oh wait, i left the other languages commented out

File: Documentation/Contributing.md [hunk 2]
```diff
@@ -200,18 +200,56 @@ First, to associate the existing template, we need to create a virtual category
     ```
     **Note that the second item in the galleries array, it has a "categoryKey". It should be match with a "key" in a virtual category.**
 
-## now that you have your workbook and settings files...
+# How to make changes (add, modify templates)
 
-1. clone the repo, if you haen't already.
+1. clone the repo, if you haven't already. if you have already, `git checkout master` and `git pull` to make sure you are up to date
```
Comment (ericajlin) [inline Documentation/Contributing.md]: nit: capitol i

File: Documentation/Contributing.md [hunk 3]
```diff
@@ -51,9 +51,9 @@ Root
 ...       
 ```
 The template galleries of the Workbooks tools are organized into categories, like Business Hypotheses, Performance, and Usage. Each category can contain many templates.
-![Image of category view](https://github.com/Microsoft/Application-Insights-Workbooks/blob/master/_assets/CategoryView.png)
+![Image of category view](..//_assets/CategoryView.png)
```
Comment (ericajlin) [inline Documentation/Contributing.md]: Since you're updating this document, new screenshots would be nice

Comment (gardnerjr) [reply Documentation/Contributing.md]: done

**Pull request merged**